packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import uniqueid from 'lodash.uniqueid';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+    * Whether or not the Switch is enabled ("ON").
+    */
+  isOn: PropTypes.bool,
+
+  /**
+    * If true, the user won't be able to toggle the switch.
+    */
+  isDisabled: PropTypes.bool,
+
+  /**
+    * Text of the label.
+    */
+  labelText: PropTypes.node.isRequired,
```

Comment:
We'll want to evaluate the name of this prop with UX. If this needs to always be a string we should change the PropType. If this can be any node type we should consider alternative naming conventions. We may choose to keep the naming convention to be more consistent with other components, but this will open the possibility of a non-text value being provided. This might be okay leaving as is, we should double check with UX. It's easier to start as a string and open up to a node type later than it is to start with a node type and switch to a string.

Reply:
adding @neilpfeiffer

Reply:
by taking other form components as reference updated prop name to [`label`](https://github.com/cerner/terra-core/blob/895c4fad7bb180e27e1919296ac90eeaa805dea1/packages/terra-switch/src/Switch.jsx#L35)